code/src/Components/SecondQuestion.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+const SecondQuestion = ({
+    days,
+    onDayChange,
+    onStepChange,
+}) => {
+    return (
+        <form>
+            <label>
+                Vilka dagar kan du?
+              <select value={days} onDayChange={onDayChange}>
```

Comment:
I tried to go throught your code, but I am not sure why you use onDayChange = {onDayChange}. It would be great if you can clarify 😄